dist/windows/strawberry.nsi.in

```diff
@@ -161,12 +178,12 @@ RequestExecutionLevel admin
 
 Function LockedListPageShow
   LockedList::AddModule /NOUNLOAD \strawberry.exe
-  LockedList::Dialog /heading "Checking for running programs:" /noprograms "No programs need to close." /searching "Searching for running programs..."
+  LockedList::Dialog /heading "${CHECKING_RUNNING_PROGRAMS}:" /noprograms "${NO_PROGRAMS_TO_CLOSE}" /searching "${SEARCHING_RUNNING_PROGRAMS}"
```

Comment:
`${variable}` for language variables are wrong everywhere. It should be `$(variable)`
`${CHECKING_RUNNING_PROGRAMS}` should be `$(CHECKING_RUNNING_PROGRAMS)`
See https://nsis.sourceforge.io/Reference/LangString

Reply:
@jonaski 

Fixed.

To avoid this iIf you can provide me in an archive the files collecetd by NSIS to create the installer I can make more tests.

Reply:
Sorry for the errors but  sometimine to write manually, you are quite tired and also use of ( { ) } and very similar and be get confusion.